core/templates/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -52,9 +52,8 @@ angular.module('oppia').directive('ckEditor4Rte', [
 
         var editable = document.querySelectorAll('.oppia-rte-resizer');
         var resize = function() {
-          var modalWidth = $('.modal-header').width() - 15;
           $('.oppia-rte-resizer').css({
-            width: (modalWidth.toString() + 'px')
+            width: '100%'
```

Comment:
Make sure to test this in all other places where we use ckeditor.
Considering the #9224 is only specific to question editor, can we add this css in the HTML file of the modal?

Reply:
From what I checked, this happens in a lot of RTEs that are present in modals (like subtopic editor, skill review material editor when creating a skill etc.) and this fix fixes all of them, so it seems like a general issue.

Reply:
Hm, I wonder why was this code written like this in the first place. Could it be related to resizing of the screen?